### ISSUE TITLE
[FEATURE] Respawn unhiding of markers.

### ISSUE THREAD

**Issue by sgtoutlaw** [EliteMasterEric/Teyvat.moe#169]
We already know the respawn time of a lot of the materials around teyvat now.  So I think that a function that allows you to enable markers to unhide themselves after a set period of time should exist.

Not sure if it is all that helpful for things that take 24 hours, but I could definitely see the use for the items that take 48 and 72 hours to respawn.

This is implemented here https://www.3839.com/hykb/hykb_tools/yuanshen/cxq/

Where if you hit that 3rd of the 4 buttons at the top of the side menu you go into a mode where you click on the resource you want to change whether it's time is tracked, and how long till the mark is cleared.

I am just not sure if being able to set your own personal timer per resource is necessary, but being able to set whether that a resource will clear itself after a set time would be helpful in resource gathering.

This would just be a nice side functionality, but I don't think it should be prioritized if it will impact map performance.  At least in my opinion.

Edit:  I wanted to add, because I don't really have any other way of doing it, my thanks for you listening to my suggestions and how well you have gone and fixed any problems I am have had.

**Comment by EliteMasterEric** [comment]
Hey @sgtoutlaw,

I am happy to say that this feature is already implemented in the map.

For supported features (I added respawn times to most of the ones that have it), navigating to About->Summary previews a count of collected marker progress. Clicking the menu button to the left side of that displays a menu containing various options. This includes locating a random uncollected marker of the specified type (great for things like Anemoculus or Chests where you're trying to find the one you haven't checked yet!), clear completion of all markers of that type, or (as suggested) clear completion of markers that have respawned.

This feature is the main reason I added the "Marked as completed" attribute, and the label on the popups as an added convenience.

I won't close this issue yet, however, because the fact you created this issue is important. What prevented you from finding the feature simply by using the app? Did you not know the Summary tab existed, or that it had options like this available?

**Comment by sgtoutlaw** [comment]
I knew the Summary tab existed, but I remember thinking it would have this function from a description of the map features, but I just didn't see it when I was using it so I thought maybe I was mistaken.  

I think that for some reason the interface isn't being obvious that this function is in effect? 

I saw I could locate items, and it would show me how many I had of that type, and also the marker would take on the current time.  It taking the time is what made me think this would be an easy to implement.

Maybe I just expected to see something that denotes the fact they have a timer, instead of just an option that is "Clear Expired".   I think perhaps the wording was misleading to me as well.  I see it and I think, "What is expired that needs to be cleared."

"Clear Respawned Resources"

Not sure if that looks the best, I just want the wording to be more obvious what it does.  

Honestly this is a little irritating, because I didn't think it existed, which is bad, but the function is something that just works without getting in the way, which is good.  

Then I can't tell whether this is actually a problem or just a me problem.  -.-


Edit:  Also I am not sure if anything ever appeared in the summary tab when using the non beta version, through a number of updates, and still doesn't.  Not sure if that is only a beta only feature or what.

**Comment by EliteMasterEric** [comment]
Yeah, currently the feature is manual, not automatic. It doesn't activate when you don't want it to.

As for the name of the option itself, my logic was that the marking had... expired or something? In retrospect it does not sound obvious. I'll look into changing the text soon.

I don't think these are "just you" problems; I'm not a gifted user interface designer, and I have a bias since I built the interface myself from scratch and therefore know how everything works. If the average user is confused, that means something should be moved or fixed.

A feature (disabled by default) that would automatically use the Clear Respawned Resources (don't think that's the best name either, hmmm...) would be nice too, I could add that as a switch in the Options menu.

Maybe a timer of some kind in the Summary tab would be cool... Find the oldest marker and display the time until it respawns. 

As for your edit, features currently only appear in the Summary tab if they are both displayed and have at least one marker completed.
